tests/tests.cpp

```diff
@@ -7,12 +7,15 @@ TEST(assert, issue11) {
   ASSERT_NO_FATAL_FAILURE(WSU::Model::StoredString storedString{});
 }
 
-TEST(assert, issue11_zeroContenets) {
+/// [issue](https://github.com/WSU-CEG-6110-4410/Words/issues/7)
+TEST(assert, issue11_zeroContenets)
```

Comment:
I see two small issues with this. 
1) Why is the variable named after issue 11 when the comment above says issue 7? Shouldn't it be named after issue 7?
2) The second half of the variable name is misspelled: issue11_zeroContenents --> issue11_zeroContents